src/xdocs/checks/header/header.xml

```diff
@@ -61,13 +61,8 @@ line 5: ////////////////////////////////////////////////////////////////////
               <td>charset</td>
               <td>Specify the character encoding to use when reading the headerFile.</td>
               <td><a href="../../property_types.html#String">String</a></td>
-              <td>
-                <code>
-                  the charset property of the parent<a href="../../config.html#Checker">
-                  Checker</a> module
-                </code>
-              </td>
-              <td>5.0</td>
+              <td><code>the charset property of the parent &lt;a href=&quot;https://checkstyle.org/config.html#Checker&quot;&gt;Checker&lt;/a&gt; module</code></td>
+              <td>4.4</td>
```

Comment:
Version number changes are not expected. Please show proof 5.0 is wrong.

Reply:
Already in PR description

Reply:
Yes, PR description shows original version is correct. Please ensure you are showing before and after on release commits.

> https://github.com/checkstyle/checkstyle/commit/326e3ae67b8cc2a1dcbcf2356b30fd9a4afa964a#diff-fa56dd39f138d19b44a0aa110150c621c7ba641aead7a198d35db46fdbc767a2R71

https://github.com/checkstyle/checkstyle/blob/326e3ae67b8cc2a1dcbcf2356b30fd9a4afa964a/pom.xml#L5
Version 5.0 .

Reply:
all since are reverted. map is updated with values from xdoc